bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,8 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+documents = {}
+
```

Comment:
thought: if the doc ids are part of the session or the embed "instance" rather than on Document itself then this global is much safer.